recipe/meta.yaml

```diff
@@ -13,9 +13,10 @@ source:
     - patches/0002-compiler-rt-Make-7.0.0-compatible-with-10.9-SDK.patch
     - patches/0003-set-cmake-paths.patch
     - patches/0004-guard-use-of-FP_XSTATE_MAGIC1-based-on-glibc-version.patch
+    - patches/D124927.patch
```

Comment:
Just look one line up, it's there already (otherwise this feedstock wouldn't build).

Reply:
It's not the upstream patch directly, but a localised revert of the change that broke things from 13->14. If you have some breakage that fails with my patch but should be fixed with the upstream one, I'm happy to swap them out of course.

Reply:
Ah sorry. You're right. I completely missed that 🤦‍♂️

Will close this out